gprofiler/perf.py

```diff
@@ -76,8 +92,16 @@ def wait_and_script(self) -> str:
         assert self._process is not None
         logger.debug(f"perf stderr: {self._process.stderr.read1(4096)}")
 
+        if self._inject_jit:
+            inject_data = Path(f"{str(perf_data)}.inject")
+            run_process(
+                [perf_path(), "inject", "--jit", "-o", str(inject_data), "-i", str(perf_data)],
```

Comment:
I guess we can't pass same file for `-i` and `-o` right ?

Reply:
Nope, it reads as a stream